src/EventStore.Core.Tests/Services/Storage/DeletingStream/when_writing_few_prepares_on_same_event_number_and_commiting_delete_on_this_version_read_index_should.cs

```diff
@@ -17,9 +17,9 @@ public class
 		protected override void WriteTestScenario() {
 			long pos;
 			string stream = "ES";
-			_streamNameIndex.GetOrAddId(stream, out var streamId, out _, out _);
+			GetOrReserve(stream, out var streamId, out pos);
 
-			var prepare1 = LogRecord.SingleWrite(_recordFactory, WriterCheckpoint.ReadNonFlushed(), // prepare1
+			var prepare1 = LogRecord.SingleWrite(_recordFactory, pos, // prepare1
```

Comment:
@timothycoleman doesn't the position change between the prepare1, prepare2, etc. writes?

Reply:
yep, the call to Writer.Write updates it between each

Reply:
ohh missed that, thanks! :)